test/e2e_node/dockershim_checkpoint_test.go

```diff
@@ -0,0 +1,225 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_node
+
+import (
+	"crypto/md5"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"regexp"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/kubernetes/pkg/api/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+const (
+	testCheckpoint          = "checkpoint-test"
+	dockershimCheckpointDir = "/var/lib/dockershim/sandbox"
```

Comment:
I left a comment on the first round of review about making this part of the test context. What do you think about that?
Here's an example: https://github.com/kubernetes/kubernetes/blob/v1.7.0-alpha.4/test/e2e/framework/test_context.go#L179
You can then access this variable: https://github.com/kubernetes/kubernetes/blob/v1.7.0-alpha.4/test/e2e_node/image_list.go#L109